fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,40 +1,40 @@
 WordPress for Android puts the power of web publishing in your pocket. It’s a website creator and so much more!
-
+ 
```

Comment:
Those changes in whitespace come from https://github.com/wordpress-mobile/WordPress-Android/pull/17341 and the fact that `android/en-US/*.txt` metadata now come from being copied from `WordPress/metadata/*.txt` files instead of being extracted from the source of `en-GB` back when we relied on a dirty hack for them before (which led to a bug if `en-GB` didn't have a translation for them). So this is not surprising.